documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -7,6 +7,8 @@ The BAMM CLI is a command line tool for the validation of Aspect models and the
 
 TIP: Download latest version: icon:download[] https://github.com/OpenManufacturingPlatform/sds-sdk/releases/download/v{sds-sdk-version}/bamm-cli-{sds-sdk-version}.jar[bamm-cli-{sds-sdk-version}.jar]
 
+NOTE: At least Java version 11 is required to successfully run the tool.
```

Comment:
Unfortunately this is not correct. The current state will run _only_ with Java 11 due to the (transitive) dependency to the Nashorn JavaScript engine, which is already deprecated in 11 and removed in the next LTS release Java 17.